python_modules/libraries/dagster-k8s/dagster_k8s/executor.py

```diff
@@ -35,25 +37,38 @@
     get_user_defined_k8s_config,
 )
 
+_K8S_EXECUTOR_CONFIG_SCHEMA = merge_dicts(
+    DagsterK8sJobConfig.config_type_job(),
+    {
+        "job_namespace": Field(StringSource, is_required=False),
+        "retries": get_retries_config(),
+        "max_concurrent": Field(
+            IntSource,
+            is_required=False,
+            description=(
+                "Limit on the number of pods that will run concurrently within the scope "
+                "of a Dagster run. Note that this limit is per run, not global."
+            ),
+        ),
+        "tag_concurrency_limits": get_tag_concurrency_limits_config(),
+        "include_k8s_config_from_run": Field(
+            BoolSource,
+            is_required=False,
+            default_value=False,
+            description=(
+                "Whether the launched step pods should include raw k8s config set on the"
+                " deployment/code location (via 'runK8sConfig' in the Helm chart) or the run"
+                " (via 'dagster-k8s/config' tags). If this is set to True, any"
+                " `dagster-k8s/config` tags on the op can still override those values."
+            ),
+        ),
+    },
```

Comment:
would we be happier in the future if we did some sort of enum describing the behavior here instead of a bool ie `base_step_k8s_config:  INHERIT | NONE`  ? I feel like all our k8s config knobs are due for a reckoning in the future so some thought for that poor soul is warranted. 

nit: i find this name a bit odd given this being runs scoped config